versioned_docs/version-22.04/developer/developer-gorgone-pull-mode.md

```diff
@@ -0,0 +1,87 @@
+---
+id: developer-gorgone-pull-mode
+title : Configuring Gorgone in pull mode
+---
+
+This procedure describes how to configure Gorgone between a distant poller and a central server.
```

Comment:
Here, pull mode offers a way to have the poller act as a client and connect to the central (which will be the server). This is useful when pollers are behind firewalls that prevent incoming traffic.